pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -54,6 +54,11 @@ self: super: {
   ghc-datasize = disableLibraryProfiling super.ghc-datasize;
   ghc-vis = disableLibraryProfiling super.ghc-vis;
 
+  # The test suite for fakedata is failing and packages hedgehog-fakedata and
+  # fakedata-quickcheck depending on it are transitevely broken so we're
+  # disabling tests here to make them work.
+  fakedata = dontCheck super.fakedata;
```

Comment:
Can you report these failing test suites upstream (if it makes sense), and put a link in this comment here?

We like to be able to keep track of when we can remove these `dontCheck` overrides.

Reply:
Yeah so, about that: I was under the impression that the tests were failing on master too but trying it again on my local it seems they're different errors so it looks like I'm doing something wrong. I'm saying that because `fakedata_1_0_1` actually builds fine and looking into the repository again it seems the test suite was fixed here:  https://github.com/fakedata-haskell/fakedata/commit/4182ff14bb0aa8513ac30f5860506b3bb96482a0

I took a look at `update-hackage.sh` and the comment in `regenerate-hackage-packages.sh` and I realize we're updating the hash of the all-cabal-hashes so the packages are up-to-date but I'm not really sure I understand how the underscored versions come and go. Given 1.0.1 will replace the de-facto version at some point this will fix itself but until then it seems to me like an assignment like the following is the way to go:
```
fakedata = self.fakedata_1_0_1;
``` 
so I just update the commit to do that, let me know if that sounds good to you too.

Reply:
We follow stackage for fakedata (since it is included) which forbids breaking changes (according to PVP). Thus we are using 0.8.0 at the moment and *can't* upgrade to 1.0.1 since this is a breaking (major even) change in between which likely causes all sorts of trouble elsewhere.

Reply:
Oh I see, then I better just keep the `dontCheck`. And as for @cdepillabout initial ask I'll just update the comment to say we can remove this when the version is 1.0.1, thanks!